x/wasm/keeper/keeper.go

```diff
@@ -411,6 +409,9 @@ func (k Keeper) execute(ctx sdk.Context, contractAddress sdk.AccAddress, caller
 	if err != nil {
 		return nil, err
 	}
+	if k.IsInactiveContract(ctx, contractAddress) {
+		return nil, sdkerrors.Wrap(types.ErrInvalid, "inactive contract")
```

Comment:
How about adding a new error type representing executing the inactive contract?

Reply:
I didn't want to modify original codes, but it would be nice to add new defined `Error` if this error code number is defined other big error code number.